docs/examples/how-tos.md

```diff
@@ -0,0 +1,396 @@
+
+```html
+<!-- The editable element in the editor's DOM structure. -->
+<div class="... ck-editor__editable ..." contenteditable="true">
+	<!-- Editable content. -->
+</div>
+```
+
+```js
+// A reference to the editor editable element in the DOM.
+const domEditableElement = document.querySelector( '.ck-editor__editable' );
+
+// Get the editor instance from the editable element.
+const editorInstance = domEditableElement.ckeditorInstance;
+
+// Use the editor instance API.
+editorInstance.setData( '<p>Hello world!<p>' );
+```
+
+### How to list all instances of the editor?
+
+By default, CKEditor 5 has no global registry of editor instances. But if necessary, such a feature can be easily implemented, as explained in this [Stack Overflow answer](https://stackoverflow.com/a/48682501/1485219).
+
+## Editor's API
+
+### How to insert some content into the editor?
+
+Because CKEditor 5 uses a custom {@link framework/architecture/editing-engine data model}, whenever you want to insert anything, you should modify the model first, which is then converted back to the view where the users input their content (called "editable"). In CKEditor 5, HTML is just one of many possible output formats. You can learn more about the ways of changing the model in the {@link framework/architecture/editing-engine#changing-the-model dedicated guide}.
+
+For example, to insert a new link at the current position, use the following snippet:
+
+```js
+editor.model.change( writer => {
+	const insertPosition = editor.model.document.selection.getFirstPosition();
+
+	writer.insertText(
+		'CKEditor 5 rocks!',
+		{ linkHref: 'https://ckeditor.com/' },
+		insertPosition
+	);
+} );
+```
+
+And to insert some plain text, you can use a slightly shorter one:
+
+```js
+editor.model.change( writer => {
+	writer.insertText(
+		'Plain text',
+		editor.model.document.selection.getFirstPosition()
+	);
+} );
+```
+
+You may have noticed that a link is represented as a text with an attribute in the editor model. See the API of the {@link module:engine/model/writer~Writer model writer} to learn about other useful methods that can help you modify the editor model.
+
+To insert some longer HTML code, you can parse it to the {@link module:engine/model/documentfragment~DocumentFragment model fragment} first and then {@link module:engine/model/model~Model#insertContent insert} it into the editor model:
+
+```js
+const content =
+	'<p>A paragraph with <a href="https://ckeditor.com">some link</a>.</p>';
+const viewFragment = editor.data.processor.toView( content );
+const modelFragment = editor.data.toModel( viewFragment );
+
+editor.model.insertContent( modelFragment );
+```
+
+### How to focus the editor?
+
+```js
+// Focus the editor.
+editor.focus();
+
+// Focus the editing area of the editor.
+editor.editing.view.focus();
+```
```

Comment:
One is just an alias for the other: https://github.com/ckeditor/ckeditor5/blob/e345faf3ca5ed2f700bc2e55176574c6b46fcdf7/packages/ckeditor5-core/src/editor/editor.ts#L550-L552

Reply:
Good to know! I copied this from https://ckeditor.com/docs/ckeditor5/latest/framework/deep-dive/ui/focus-tracking.html#how-to-focus-the-editor

Does this one also needs to be fixed?